timify-ui/src/components/Rating/Rating.jsx

```diff
@@ -58,30 +58,30 @@ const Rate = ({episodeId}) => {
 
 
 	return (
-		<div className="stars">
+		<Container>
 			{[...Array(5)].map((item, index) => {
 				const givenRating = index + 1;
 				return (
 					<label key={index}>
-						<div className="radio"
+						<Radio
 							type="radio"
 							value={givenRating}
 							onClick={() => handleRatingChange(givenRating)}
-						>
-						</div>
-						<div className="rating">
+						/>
+						
+							<Rating>
 							<FaStar
 								color={
 									givenRating < rate || givenRating === rate
 										? "ffae42"
 										: "rgb(192,192,192)"
```

Comment:
is it possible to save these colors as variables somehow so the names are more descriptive to us?